src/app/services/organization-redirect-guard.service.ts

```diff
@@ -0,0 +1,36 @@
+import { Injectable } from "@angular/core";
+import { ActivatedRouteSnapshot, CanActivate, Router } from "@angular/router";
+
+import { I18nService } from "jslib-common/abstractions/i18n.service";
+import { OrganizationService } from "jslib-common/abstractions/organization.service";
+import { PlatformUtilsService } from "jslib-common/abstractions/platformUtils.service";
+import { Permissions } from "jslib-common/enums/permissions";
+import { Utils } from "jslib-common/misc/utils";
+
+@Injectable()
+export class OrganizationRedirectGuardService implements CanActivate {
```

Comment:
It's not obvious at a glance that this guard checks if any organization matches the conditions. Maybe we should rename it or add a description?

Reply:
Renamed to `RedirectToOrgAdminGuardService` and added a js-doc comment.
Also raised in Slack whether we can drop `Service` from these names because it's messy.